src/renderer/const.ts

```diff
@@ -235,11 +239,13 @@ export const ASGARDEX_SWAP_IDENTIFIER = 999
 
 export const RECOVERY_TOOL_URL: Record<Network, string> = {
   testnet: 'https://testnet.thorswap.finance/pending',
+  stagenet: 'https://stagenet.thorswap.finance/pending',
```

Comment:
@veado Wanted confirmation as to whether this should be stagenet specific or the mainnet value.

Reply:
It seems https://stagenet.thorswap.finance/pending is accessible, so let's use it.